Java/parkservices/src/main/java/com/amazon/randomcutforest/extendedrandomcutforest/AnomalyDescriptor.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.randomcutforest.extendedrandomcutforest;
```

Comment:
`randomcutforest.extendedrandomcutforest` is a mouthful. Why not `randomcutforest.parkservices`. If we feel that's not descriptive enough, then how about just `randomcutforest.extended`? (but if "parkservices" isn't a good java package name, is it still a good maven module name?)

Reply:
No, we can have randomcutforest.parkservices. It flattens the structure -- but maybe AnomalyDescription is something all of parkservices aspire to adhere to. Changing.